tests/test_generate.py

```diff
@@ -277,6 +293,44 @@ def test_generate(yaml_dir, tmpdir):
     t7 = html_blob.updated
     assert t6 != t7
 
+    # Upload new blob, build only latest, and verify only latest is updated.
+    new_metadata = "docs.metadata.newer"
+    # Swap to newer metadata to upload newer version of tarball.
+    swap_file(yaml_dir, yaml_dir / "docs.metadata", yaml_dir / new_metadata)
+    latest_html_blob_name = "python-doc-pipeline-test-2.1.2.tar.gz"
```

Comment:
Nit: any particular reason to have this in between the swap-upload-swap?

Reply:
Nay, I'll move it up.